### DISCUSSION TITLE
I can't login to my old account and delete a custom domain on that account? What do I do?

### DISCUSSION THREAD

**Discussion by InteldaTechnology** [community/community#129818] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I can't login to **my _old_ account** and delete a `custom domain` on that account?
_> `What do I do?`_

**Comment by LiteBrite82** [comment]
Hi @InteldaTechnology ! 

The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Comment by vishveshjain** [comment]
First remove custom domain then clear cache or disable cache for that particular website.
